Projects/19-Captsone_01/README.md

```diff
@@ -0,0 +1,830 @@
+# Insights into the I-SPY clinical trial
```

Comment:
Are you generating https://jcardenasrdz.github.io/Insights-into-the-I-SPY-clinical-trial/ from this file? Asking for a friend.